src/Sonarr.Http/Extensions/NancyJsonSerializer.cs

```diff
@@ -1,22 +1,23 @@
 ﻿using System.Collections.Generic;
 using System.IO;
 using Nancy;
+using Nancy.Responses.Negotiation;
 using NzbDrone.Common.Serializer;
 
 namespace Sonarr.Http.Extensions
 {
     public class NancyJsonSerializer : ISerializer
     {
-        public bool CanSerialize(string contentType)
+        public bool CanSerialize(MediaRange contentType)
         {
             return true;
```

Comment:
We need to check the content type here, coz in my tests we return json when xml is explicitly requested, and even return Content-Type: application/xml... with a json body.